Formula/yb-voyager.rb

```diff
@@ -0,0 +1,22 @@
+class YbVoyager < Formula
```

Comment:
Didn't we plan to create version specific Formula files and then make the `yb-voyager.rb` a symlink to the latest one?

Reply:
Okay I thought that was for future use. I will create `beta.0` specific file and add a symlink to it.

Reply:
The naming format for versioned formulae doesn't
support anything like `beta.0`. The class has to be named something like `YbVoyagerATbeta.0` but this gives an error while auditing the formula.

Reply:
The version has to be only digits and dots after the 'AT' part in the class name.

Reply:
The naming format for versioned formula classes doesn't support anything like 'beta.0'. I will have to create a class like `YbVoyagerATbeta0`. But this gives an error while auditing the formula. 
There should be only digits after the 'AT' part in the class name is what I could infer from the errors. Any proper guide on naming is not available.